src/rgw/rgw_sal_motr.cc

```diff
@@ -2309,7 +2328,7 @@ int MotrObject::copy_object_same_zone(RGWObjectCtx& obj_ctx,
   bufferlist bl;
   rc = read_op->get_attr(dpp, RGW_ATTR_ETAG, bl, y);
   if (rc < 0){
-    ldpp_dout(dpp, 20) << "ERROR: read op iterate failed rc=" << rc << dendl;
+    ldpp_dout(dpp, 20) <<__func__<< "ERROR: read op for etag failed rc=" << rc << dendl;
```

Comment:
remove __func__ from log, its not followed elsewhere

Reply:
keeping it for ease of debugging